test/functional/test_framework/messages.py

```diff
@@ -1090,10 +1090,10 @@ class msg_verack:
     msgtype = b"verack"
 
     def __init__(self):
-        pass
+        raise NotImplementedError()
 
     def deserialize(self, f):
-        pass
+        raise NotImplementedError()
```

Comment:
These are stubs that gracefully pass and the same goes for every other change to this file. Needs to be documented, not regressed into throwing exceptions.